Merge dev into master (#130)
satisfactorymodding/Documentation: modules/ROOT/pages/CommunityResources/AssetToolkit.adoc

Take the `AssetGenerator` folder out of the files you downloaded
and move it to your modding project's `Plugins` folder.

This plugin will stay in the editor only.

=== Installing the Asset Dumper

Take the `AssetDumper` folder out of the files you downloaded
and move it to your modding project's `Plugins` folder.

This plugin will be built as a mod to install in the game.

=== Compiling

> After installing the two plugins in the editor,

Reviewer: Both plugins must be put in the editor. One of them is used as an editor plugin, the other needs to be built for the game via alpakit.

What wording would better reflect this? Maybe "After putting both plugins' folders in your Plugins folder" but that says 'plugin' and 'folder' too many times.

> After installing the two plugins in the editor,

Author: I think "putting both plugins in your Plugins folder" is alright

> After installing the two plugins in the editor,

Reviewer: ```suggestion
After putting both plugins in your project's Plugins folder,
```

====
Asset dumping can also be performed automatically via the command line.
Check out the
https://github.com/mircearoata/satisfactory-modding-actions/blob/master/.github/workflows/updateSML.yml#L209[CI scripts],
or contact us on the discord for more info.
====

=== Importing Dumped Assets to the Editor

You can expect a Content folder with all assets imported to take up about 2 GB.

> For comparison, a Content folder with placeholders takes up 366s MB.

Reviewer: ```suggestion
For comparison, a Content folder with placeholders takes up 366 MB.
```

I probably hit `s` and not `ctrl+s` at some point. Placeholders folder doesn't get _that_ big

> For comparison, a Content folder with placeholders takes up 366s MB.

Author: Oop, I didn't read properly and thought this was talking about a complete project. My bad

> For comparison, a Content folder with placeholders takes up 366s MB.

Reviewer: Had to reopen so I could add this commit to the batch
